testdata/ruby/obj/api/toc.yml

```diff
@@ -5,6 +5,8 @@
     items:
     - name: Getting started
       href: index.md
+    - name: License
+      href: LICENSE.md
```

Comment:
Unrelated to this PR, but it's surprising to me to include the license?

Reply:
It _is_ intentional. The license is included in the standard Ruby reference documentation for our libraries, and currently Ruby's cloud-rad strategy is to mirror the standard documentation layout. That said, if it's an issue, we can include logic to omit it.